taxonomy-editor-frontend/src/pages/root-nodes/index.tsx

```diff
@@ -49,13 +53,53 @@ const RootNodes = ({
     useState(false);
 
   const baseUrl = createBaseURL(taxonomyName, branchName);
+  const projectInfoUrl = `${baseUrl}project`;
+  const rootNodesUrl = `${baseUrl}rootentries`;
+
+  const {
+    data: info,
+    isPending: infoPending,
+    isError: infoIsError,
+    error: infoError,
+  } = useQuery<ProjectInfoAPIResponse>({
+    queryKey: [projectInfoUrl],
+    queryFn: async () => {
+      const response = await fetch(projectInfoUrl);
+      if (!response.ok) {
+        throw new Error("Failed to fetch project info");
+      }
+      return response.json();
+    },
+    refetchInterval: (d) => {
+      return d.state.status === "success" &&
+        d.state.data?.status === ProjectStatus.LOADING
+        ? 1000
+        : undefined;
```

Comment:
I think it should be returning `false` according to the documentation

Reply:
you're right it's number or false